### ISSUE TITLE
add link to VPAT statement on accessibility-info page

### ISSUE THREAD

**Issue by tkimnguyen** [plone/Products.CMFPlone#2408]
The VPAT statement https://www.state.gov/m/irm/impact/126340.htm is part of the accessibility compliance story for Plone (though perhaps US specific).

We now have one at https://plone.com/vpat-accessibility-statement/view (linked to from https://plone.com/accessibility-info). It should be similarly linked to from the default accessibility-info page on all Plone sites.

**Comment by tkimnguyen** [comment]
![screen shot 2018-05-13 at 11 15 02 am](https://user-images.githubusercontent.com/815418/39969275-f0ebce16-569e-11e8-9c2a-2892c6976438.jpg)

**Comment by laulaz** [comment]
@tkimnguyen the link to VPAT statement (https://www.state.gov/m/irm/impact/126340.htm) used in https://github.com/plone/Products.CMFPlone/blob/2ca5b4227187215eaa58b3a91d66e3cdccb6256b/Products/CMFPlone/browser/templates/accessibility-info.pt#L71 is not available anymore  😞

**Comment by gforcada** [comment]
Maybe is this one https://www.state.gov/section-508-accessibility-statement/ ?

**Comment by tkimnguyen** [comment]
I don't recall what was at the original link now. Let me check with my "contact" :)

**Comment by tkimnguyen** [comment]
Ah, here is an archived version of that page: https://web.archive.org/web/20150706073343/https://www.state.gov/m/irm/impact/126340.htm

**Comment by tkimnguyen** [comment]
Ah, and then I found mention of the State Department's archive site, where I found this search results page for "VPAT": https://findit.state.gov/search?query=vpat&affiliate=dos_2009_2017_stategov&search=

**Comment by tkimnguyen** [comment]
we should probably link to this https://2009-2017.state.gov/documents/organization/126552.pdf

**Comment by yurj** [comment]
what is the state of this issue? The actual link lead to nowhere, maybe we should put this link: https://www.section508.gov/sell/vpat/ ?

**Comment by jensens** [comment]
Still relevant for Plone 6. VPAT link on accessibility-info page is broken/outdated. Needs updated URL.
